kernel/aster-nix/src/fs/path/dentry.rs

```diff
@@ -667,6 +678,18 @@ impl Dentry {
         Ok(())
     }
 
+    /// Move process's root from this mount node to another.
+    pub(super) fn move_root(&self, mount_node: &Arc<MountNode>, process: &Arc<Process>) {
```

Comment:
How to ensure that the process's current root is `self` ? If the process's current root is not `self`, will this cause any unexpected behavior?